packages/ui/src/components/language-selector/language-dialog.tsx

```diff
@@ -3,11 +3,12 @@ import { FC, useEffect, useState } from 'react'
 import { Dialog, Icon } from '@/components'
 import { cn } from '@utils/cn'
 
-import { LanguageCode, LanguageDialogProps, languages } from './types'
+import { LanguageCode, LanguageDialogProps } from './types'
```

Comment:
can we also delete the `languages` object if it's not being used anymore

Reply:
sure